src/MIDebugEngine/Engine.Impl/DebuggedThread.cs

```diff
@@ -46,6 +46,30 @@ internal class ThreadCache
         private static uint s_targetId;
         private const string c_defaultGroupId = "i1";  // gdb's default group id, also used for any process without group ids
 
+        private List<DebuggedThread> deadThreads
```

Comment:
>d [](start = 37, length = 1)

nit: the convention in this code base is to capitalize the first letter of properties.